delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/MainSuite.scala

```diff
@@ -0,0 +1,129 @@
+package ch.epfl.bluebrain.nexus.delta
+
+import akka.http.scaladsl.server.Route
+import cats.effect.Resource
+import ch.epfl.bluebrain.nexus.delta.plugin.PluginsLoader.PluginLoaderConfig
+import ch.epfl.bluebrain.nexus.delta.sdk.plugin.PluginDef
+import ch.epfl.bluebrain.nexus.delta.wiring.DeltaModule
+import ch.epfl.bluebrain.nexus.testkit.bio.{BioSuite, ResourceFixture}
+import ch.epfl.bluebrain.nexus.testkit.elasticsearch.ElasticSearchContainer
+import ch.epfl.bluebrain.nexus.testkit.postgres.Doobie._
+import ch.epfl.bluebrain.nexus.testkit.postgres.PostgresContainer
+import com.typesafe.config.impl.ConfigImpl
+import izumi.distage.model.definition.{Module, ModuleDef}
+import izumi.distage.model.plan.Roots
+import izumi.distage.planning.solver.PlanVerifier
+import monix.bio.Task
+import munit.AnyFixture
+
+import java.nio.file.{Files, Paths}
+
+/**
+  * Test class that allows to check that across core and plugins:
+  *   - Plugins have been successfully assembled
+  *   - HOCON configuration files match their classes counterpart
+  *   - Distage wiring is valid
+  */
+class MainSuite extends BioSuite with MainSuite.Fixture {
```

Comment:
`MainSpec` rewritten as a MUnit test, it is easier to work with `cats.effect.Resource` than `BeforeAndAfterAll` and I managed to fix the problem related to Elasticsearch this way